src/components/Content/index.tsx

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable react/function-component-definition */
+import { FC } from 'react';
+import Header from 'components/Header';
+import Widgets from 'components/Widgets';
+// styles
+import s from './Content.module.scss';
+
+const Content: FC = () => (
+	<div className={s.content}>
+		<div className={s.header}>
```

Comment:
u no need wrapper above the header here. pass it inside component

Reply:
done